contrib/example-backend/controllers/clusterbinding/clusterbinding_controller.go

```diff
@@ -199,13 +187,21 @@ func mapServiceExportToClusterBinding(ctx context.Context, obj client.Object) []
 
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
-func (r *ClusterBindingReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+func (r *ClusterBindingReconciler) Reconcile(ctx context.Context, req mcreconcile.Request) (ctrl.Result, error) {
 	logger := log.FromContext(ctx)
 	logger.Info("Reconciling ClusterBinding", "namespace", req.Namespace, "name", req.Name)
```

Comment:
Can we instead try `log.Info("Reconciling ClusterBinding", "request", req)`, so the cluster is also included?

Also, can we use a scoped logger like

```go
logger := log.FromContext(ctx).With("request", req)
logger.Info("Reconciling ClusterBinding")
```

so that all following log statements are tagged accordingly?